inline.h

```diff
@@ -3430,6 +3430,13 @@ Perl_cop_file_avn(pTHX_ const COP *cop) {
 
 #endif
 
+PERL_STATIC_INLINE PADNAME *
+Perl_padname_refcnt_inc(PADNAME *pn)
```

Comment:
Sudden wondering whether this should be named `S_` instead of `Perl_`, as it's only here to make one of the macros work.

Reply:
Eh; most of the rest of them are `Perl_...` and those seem fine. I'll leave it as is.